src/SignalRServiceExtension/TriggerBindings/SignalRTriggerBindingProvider.cs

```diff
@@ -39,6 +41,12 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
             {
                 return Task.FromResult<ITriggerBinding>(null);
             }
+
+            if (!_enabled)
```

Comment:
Add a UT to cover the case to cover the negative case?

Reply:
Added